geanyminiscript/src/gms_gui.c

```diff
@@ -26,7 +26,7 @@
  * \brief it is the graphical user interface of the geany miniscript plugin
  */
 #include    "config.h"
-#include    "geany.h"
+#include    "geanyplugin.h"
 
```

Comment:
This change fixes
  CC       gms_gui.lo
In file included from gms_gui.c:57:0:
gms.h:30:8: error: unknown type name 'GeanyFunctions'
 extern GeanyFunctions   *geany_functions;

+#include "geanyfunctions.h" would also work but geanyplugin.h is recommended anyway

Reply:
I do not get that error if I build from current master.